src/libs/style.ts

```diff
@@ -237,7 +237,7 @@ export const dashboard = {
     textTransform: 'uppercase',
     color: colors.dark(),
   },
-};
+} as const satisfies Record<string, CSSProperties>;
```

Comment:
Unfortunately IntelliJ isn't too happy with this, but perhaps I need to update its elsint version.